build/cwl-wrappers/cmo-marianas/0.0.0/outputs.yaml

```diff
@@ -0,0 +1,30 @@
+outputs:
+  processed_fastq_1:
+    type: File
+    outputBinding:
+      glob: $(inputs.r1_fastq)
+
+  processed_fastq_2:
+    type: File
+    outputBinding:
+      glob: $(inputs.r2_fastq)
+
+#  composite_umi_frequencies:
+#    type: File
+#    outputBinding:
+#      glob: composite-umi-frequencies.txt
+
+  info:
+    type: File
+    outputBinding:
+      glob: info.txt
+
+#   sample_sheet:
+#     type: File
+#     outputBinding:
+#       glob: SampleSheet.csv
+
+  umi_frequencies:
+    type: File
+    outputBinding:
+      glob: umi-frequencies.txt
```

Comment:
This comment is related to all of the template files that get used to generate the final `.cwl`

After running `./build-cwl.sh` there are some spacing issues and I had to manually edit the final cwl to get it correctly formatted.

Reply:
Can reproduce that output if needed